### ISSUE TITLE
Bug bounty

### ISSUE THREAD

**Issue by benmccann** [Exiv2/exiv2#2326]
##### **Is your feature request related to a problem?**

I'd like to see one of the issues in this repository fixed. I don't currently possess the skill set to do that myself, but would be happy to sponsor the work. I realize developer time is expensive, and am willing to compensate accordingly.

##### **Describe the solution you would like**

If anyone would be interested in fixing an issue here, please let me know. I'd be happy to either renumerate the developer responsible for fixing the issue directly or to donate to a fund to support the project's expenses.

##### **Describe alternatives you have considered**

I could teach myself C++
I could find someone who knows C++, but is unfamiliar with this codebase and pay them to submit a PR

##### **Additional context**

The particular issue I'm interested in seeing fixed is https://github.com/Exiv2/exiv2/issues/2237 as exiv2 seems not to work with videos taken by Android phones, which is a large fraction of videos taken today. The software I use to manage my personal photo collection utilizes exiv2 and is thus unable to handle metadata from a large fraction of videos I have taken

**Comment by benmccann** [comment]
Closing as it seems video support has been dropped from the library

**Comment by hassec** [comment]
@benmccann first off, I'd like to say thank you for offering to sponsor someone to fix this issue, that is very generous!  

Exiv2 unfortunately, so far, isn't setup to receive such funds, and I've never looked into how such funding works. And right now exiv2 is pretty much without a full time maintainer, so I'm not sure what we could reasonably do with the donation.

That said, I do hope that we might find enough time to reinstate video support, given that it seems to be a quite frequently requested feature. 
IMHO, it would be nice to at least offer a best effort support of the most common formats.

**Comment by benmccann** [comment]
In terms of accepting donations, many projects I've used and contributed to have used https://opencollective.com/  If there are no developers able to work on the project on a paid basis, it could potentially be helpful for covering hosting costs, etc.

Anyway, while I've closed this issue, consider it a standing offer. I'd be happy to support the project financially if it's able to help the project with read support for the most common video formats. Thanks for helping to support this library!
